src/AppBundle/Resources/views/Admin/Client/Report/partials/_assets-and-debts.html.twig

```diff
@@ -2,7 +2,7 @@
 {% trans_default_domain translationDomain %}
 {% set page = 'checklistPage' %}
 
-<h3 class="heading-medium underline">{{ (page ~ '.heading.assetsAndDebts') | trans }}</h3>
+<h3 class="heading-medium underline" id="assetsAndDebts">{{ (page ~ '.heading.assetsAndDebts') | trans }}</h3>
```

Comment:
just curiosity. are ID needed for this PR ?

Reply:
Just using the IDs to jump to the right place